src/pre_defined_classes/simple_programs_taking_numeric_user_input/medium/q6/Intelligence.java

```diff
@@ -0,0 +1,26 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.medium.q6;
+/*
+A computer science teacher would like to ask the students about the intelligence of computers versus humans.
+Make a program that asks the user if the user thinks that computers are smarter than humans.
+Generates a random percentage representing how right they are and display the percentage to the users
```

Comment:
The start of this sentence does not make sense

Reply:
Should be something like `The program should generate...` or `Then, generate a random ...`